### PR TITLE
launchctl: add page

### PR DESCRIPTION
**Pull request by waldyrious**
Built with info from several tutorials online. Unfortunately I seem to have discarded the links :(

### CODE REVIEW

File: pages/osx/launchctl.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Activate a system-wide daemon to be loaded whenever the system boots up (even if no user logs in):
+
+`sudo launchctl load /Library/LaunchDaemons/{{system_daemon}}.plist`
+
+- Show all loaded agents/daemons, with the PID if the process they specify is currently running, and the exit code returned the last time they ran:
+
+`launchctl list`
+
+- Unload a currently loaded agent, e.g. to make changes (note: the plist is automatically loaded into `launchd` after a reboot and/or logging in):
```
Comment (agnivade) [inline pages/osx/launchctl.md]: Doesn't "the plist file" sound better ? plist is just the extension right ?

And also, "e.g. to make changes" seems obvious to me. Why else would you unload an agent ? Either to stop it or to make changes.

Comment (waldyrious) [inline pages/osx/launchctl.md]: It may be better to make it explicit that you must (I suppose?) unload an agent before changing the plist file, e.g. in case it is locked by the system, or would cause conflicts, or the changes wouldn't apply without unload/reload. Actually, I don't know which of these (if any) is the case, I just took up the wording from the tutorials I read. I would guess that it's the latter meaning, so we could reword this to make it clearer. Do you agree that the last option seems more plausible?

As for "the plist file", agreed.

Comment (agnivade) [inline pages/osx/launchctl.md]: >  or the changes wouldn't apply without unload/reload. 

Yes, that might be the case. I feel its obvious. But sure, go ahead. Its always good to make it clearer.

Comment (waldyrious) [inline pages/osx/launchctl.md]: I can't think of a way to be more explicit without being more verbose... I guess I'll leave it as it is for now, and if a better wording comes up, we can change it :)

**Comment by agnivade** [comment]
LGTM.

**Comment by waldyrious** [comment]
Great. I'll add my own LGTM as I'm a maintainer, so we can get this one behind us :P
